Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# nightelf magical eyes

## Changelog:
- Added golden eyes for Night Elf race that have natural magic traits (Otherworldly, Sorcerous, Magical, Magi) 


## Developer changelog:
- Tied magic_good traits to night elf trait in 00_trait_modifiers.txt


# How to test:
If using the Ruler editor, assign magic traits (Otherworldly, Sorcerous, Magical, Magi) to character. Eyes will not immediately change until you accept your character- the night elf trait doesn't get assigned to a character until outside of ruler editor. 
To make a night elf, make sure character's culture is in the sub group "Darnassian", which is under the Elven culture. This also forces a character to gain the night elf trait (despite their appearance or apparent ethnicity.)
Otherwise, any night elf in game already should already inherit these eyes. Later gameplay needs testing to confirm offspring heritability, how culture mixing works out, etc.
